plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/core/connection/Connection.java

```diff
@@ -382,16 +393,16 @@ public String toString() {
 			}
 			return client.get(resource.getKind(), resource.getName(), resource.getNamespace());
 		} catch (UnauthorizedException e) {
-			return retryGet("Unauthorized.  Trying to reauthenticate", e, resource);
+			return retryGet("Unauthorized.  Trying to reauthenticate", e, resource.getKind(), resource.getName(), resource.getNamespace());
```

Comment:
trying to understand: how is this related to the inability to retrieve istags? Did the backend change?